docs/guides/arm_virtual_hardware_ble-wifi-commissioning.md

```diff
@@ -0,0 +1,216 @@
+
+## Lighting-app Virtual instance
+
+1. Log into https://app.avh.arm.com/login using your AVH account.
+
+2. In the Device tab, click the Create Device button.
+
+![AVH_create_device](images/AVH_create_device.png)
+
+3. Then click on the Raspberry Pi 4 board to select it.
+
+![AVH_select1](images/AVH_select1.png)
+
+4. Click the Next button to continue
+
+![AVH_select2](images/AVH_select2.png)
+
+5. Click the “browse” link on right-hand side and select the Ubuntu Server 22.04 image from the pre-requisites. 
+
+![AVH_configure1](images/AVH_configure1.png)
+
+6. Once uploaded click Next
+
+![AVH_configure2](images/AVH_configure2.png)
+
+7. Enter a device name and click the Create Device button. Any name is fine. We used `ubuntu-lighting`
+
+![AVH_launch_device](images/AVH_launch_device.png)
+
+8. Wait for the virtual device to be created and start up. Click the Console tab to interact with the virtual board.
+
+![AVH_console1](images/AVH_console1.png)
+
+9. Use username: ubuntu and password: ubuntu to log into the console and change your password.
+
+![AVH_console2](images/AVH_console2.png)
+
+10. As per the [Building Matter - Installing prerequisites on Raspberry Pi 4 section](https://github.com/project-chip/connectedhomeip/blob/master/docs/guides/BUILDING.md#installing-prerequisites-on-raspberry-pi-4), edit /etc/systemd/system/dbus-fi.w1.wpa_supplicant1.service with sudo and a text editor of your choice. Change the ExecStart line to  
+
+```console
+ExecStart=/sbin/wpa_supplicant -u -s -i wlan0 -c /etc/wpa_supplicant/wpa_supplicant.conf`
+```
+
+11. Create the file `/etc/wpa_supplicant/wpa_supplicant.conf` with sudo and a text editor of your choice. Add the following:
+
+```console
+ctrl_interface=DIR=/run/wpa_supplicant
+update_config=1
+```
+
+12. Restart wpa_supplicant:
+
+```console
+
+sudo systemctl restart wpa_supplicant.service
+sudo systemctl daemon-reload
+```
+13. Install dependencies
+
+```console
+sudo apt-get update
+sudo apt-get -y install git gcc g++ python3 pkg-config libssl-dev libdbus-1-dev libglib2.0-dev libavahi-client-dev ninja-build python3-venv python3-dev python3-pip unzip libgirepository1.0-dev libcairo2-dev libreadline-dev 
+```
+
+14. Press tab to select “Ok” when presented with the following prompt, then press enter.
+
+![AVH_console3](images/AVH_console3.png)
+
+15. Clone the Matter repo and setup sub modules
+
+```console
+git clone https://github.com/project-chip/connectedhomeip.git 
+cd connectedhomeip
+./scripts/checkout_submodules.py --shallow --platform linux
+```
+
+16. Setup development environment (takes 5-10 minutes).
+
+```console
+./scripts/build/gn_bootstrap.sh
+source scripts/activate.sh
+```
+
+17.	Build lighting-app example.
+
+```console
+cd examples/lighting-app/linux
+gn gen out/debug
+ninja -C out/debug
+```
+
+18. Disable the Ethernet interface
+
+```console
+sudo ip link set dev eth0 down
+```
+
+19. Run lighting-app example.
+
+```console
+./out/debug/chip-lighting-app --wifi
+```
```

Comment:
Having the wifi/ble is cool but do we have the equivalent with just Ethernet IP ?